nuxeo-common/src/main/java/org/nuxeo/common/utils/TextTemplate.java

```diff
@@ -426,6 +426,15 @@ public void write(char[] cbuf) throws IOException {
                 if (filename.endsWith(ext)) {
                     processAsFreemarker = true;
                     out = new File(out.getCanonicalPath().replaceAll("\\.*" + Pattern.quote(ext) + "$", ""));
+                    if (filename.equals("." + ext.toLowerCase())) {
+                        throw new IOException("Template file : '"
+                                + in.getAbsolutePath()
+                                + "' cannot be parsed. Launcher had to stop, please fix this problem."
+                                + System.getProperty("line.separator")
```

Comment:
Don't use `line.separator` in messages, just `\n` (the display layer will translate it to the platform one if needed).
But here it's an exception message and it's better to not have newlines at all.